libindi/libs/indibase/defaultdevice.h

```diff
@@ -463,6 +463,8 @@ class INDI::DefaultDevice : public INDI::BaseDevice
         }
 
         void setDefaultPollingPeriod(uint32_t period);
+        void setMinimumPollingPeriod(uint32_t period);
```

Comment:
Wouldn't be better to have one function?

setPollingPeriodRange(min, max) ?

Reply:
Yes, that's better, done.